src/main/flight/pid.c

```diff
@@ -1102,6 +1086,10 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
 
         // -----calculate error rate
         errorRate = currentPidSetpoint - gyro.gyroADCf[axis]; // r - y
+        {
+        	const float nonLinearBoost = featheredPIDboost(errorRate);
+        	errorRate *= nonLinearBoost;
```

Comment:
Does this multiply the error by our boost? Would we not want to use a += here instead?

Aside from this now that you have commented everything is looking good :)